lib/ide/src/todo/executor.rs

```diff
@@ -0,0 +1,108 @@
+//! TODO [mwu] This module is still provisional work in progress. Currently
```

Comment:
Just one request - when this module will be done, please ping us durinmg review to do review of the whole file, not only chagnes. I'm skipping this file now.